OpenRA.Mods.Common/Scripting/Properties/AircraftProperties.cs

```diff
@@ -16,12 +16,13 @@
 
 namespace OpenRA.Mods.Common.Scripting
 {
+	[ExposedForAIScript]
```

Comment:
It would be better if you could tag this at the method level instead of splitting the classes.  You could then define parameters for individually exposing get/set on properties, solving the resource issue.